test/unit_tests/utils/test_state.py

```diff
@@ -95,7 +95,7 @@ def test_inject_args(generic_product_1, generic_product_type_1) -> None:
     generic_sub.pb_2.rt_2 = 42
     generic_sub.pb_2.rt_3 = "test2"
 
-    generic_sub = change_lifecycle(generic_sub, SubscriptionLifecycle.ACTIVE)
+    generic_sub = SubscriptionModel.from_other_lifecycle(generic_sub, SubscriptionLifecycle.ACTIVE)
```

Comment:
Why not import the 'from_other_lifecycle'?

Reply:
Its a classmethod.